lib/api/controllers/securityController.js

```diff
@@ -711,12 +690,25 @@ class SecurityController {
     assertHasId(request);
     assertHasBody(request);
 
-    const
-      updateMethod = this.kuzzle.pluginsManager.getStrategyMethod(request.input.args.strategy, 'update'),
-      validateMethod = this.kuzzle.pluginsManager.getStrategyMethod(request.input.args.strategy, 'validate');
+    const 
+      id = request.input.resource._id,
+      strategy = request.input.args.strategy;
+
+    return this.kuzzle.repositories.user.load(id)
+      .then(user => {
+        if (user === null) {
+          throw new BadRequestError(`Cannot create credentials: unknown kuid "${id}"`);
```

Comment:
![bart](http://www.addletters.com/pictures/bart-simpson-generator/bart-simpson-generator.php?line=I+will+not+copy-paste+code)